crates/factors/tests/smoke.rs

```diff
@@ -20,6 +21,7 @@ struct Factors {
     outbound_networking: OutboundNetworkingFactor,
     outbound_http: OutboundHttpFactor,
     key_value: KeyValueFactor,
+    outbound_pg: OutboundPgFactor,
```

Comment:
@lann I think we should decide here and now if adding new factors to the smoke test is necessary. If the point of the smoke test is to make sure `#[derive(Factors)]` works, I think we've already accomplished that, and this change shouldn't be necessary.

Reply:
Agreed - I don't think we need to keep updating this test.